src/Uno.Extensions.Localization.UI/LocalizationSettings.cs

```diff
@@ -1,6 +1,12 @@
 ﻿namespace Uno.Extensions.Localization;
 
+/// <summary>
+/// Contains settings for the localization feature.
+/// </summary>
 public record LocalizationSettings
```

Comment:
I think this could be made internal too

Reply:
Addressed with 75f9665a96b59f6b97c160a99450674ab620a73a